config.json

```diff
@@ -589,6 +589,18 @@
         "arrays",
         "bitwise_operations"
       ]
+    },
+    {
+      "slug": "protein-translation",
+      "uuid": "fb73e822-4831-4d33-a049-44a514bfa46a",
+      "core": false,
+      "unlocked_by": null,
```

Comment:
Unlocked by reverse-string maybe?

Several of the exercises are unlocked by nothing, but we're trying not to do that anymore.